go/ql/lib/semmle/go/Expr.qll

```diff
@@ -990,8 +990,11 @@ class FuncTypeExpr extends @functypeexpr, TypeExpr, ScopeNode, FieldParent {
   /** Gets a parameter of this function type. */
   ParameterDecl getAParameterDecl() { result = this.getParameterDecl(_) }
 
+  /** Gets the number of parameter declarations of this function type. */
+  int getNumParameterDecls() { result = count(this.getAParameterDecl()) }
+
   /** Gets the number of parameters of this function type. */
-  int getNumParameter() { result = count(this.getAParameterDecl()) }
+  int getNumParameter() { result = count(this.getAParameterDecl().getANameExpr()) }
```

Comment:
I'd document specifically what a parameter decl is there, and update the qldoc of `getAParameterDecl`. I presume this is `f(x, y, z int)` where that's one decl declaring three parameters?

Reply:
I have now given examples.